### ISSUE TITLE
ABOUT TIMBER TRAILER

### ISSUE THREAD

**Issue by MHMTALP** [loki79uk/FS25_UniversalAutoload#129]
Hello 

First of all, thank you very much for this mod.

I expect support from you for this error.

It is no longer possible to load the trailer again.
"Unable to load any more Items of this size."

The timber transport trailer continues to give the above error after purchasing 2 pieces of timber.
![Uploading Ekran Alıntısı.JPG…]()

**Comment by jufcar** [comment]
Please try again with the latest version of the mod, there have been some fixes for the problem you mentioned.

**Comment by loki79uk** [comment]
Did you resize the loading volume in the shop?  Most of the log trailers are too small.  See e.g.: https://github.com/loki79uk/FS25_UniversalAutoload/issues/125

**Comment by 17goesfast** [comment]
I used the HW80 Wood Trailer By: Kamilos0397 and the base game little wood trailer and I found the load box needed to be really big.  Both worked flawlessly once the load box was made over sized.

**Comment by MHMTALP** [comment]
I used the original log carrying mode of the PITTS LT40-8L game and tried it with the latest version. It still takes 2 pieces of timber and Unable to load any more Items of this gives you this error. As can be seen in the photo I sent, there are two timbers.
![Ekran Alıntısıddd](https://github.com/user-attachments/assets/358f4366-683f-4881-a630-823965f4e6fb)

**Comment by loki79uk** [comment]
As you can see in the photo you sent, the loading area is too small, as suggested in my first reply to this bug, please resize in the shop.  See instructions in the other bug of how to do this:

> Did you resize the loading volume in the shop? Most of the log trailers are too small. See e.g.: #125

**Comment by MHMTALP** [comment]
THANK YOU, THE PROBLEM IS SOLVED

**Comment by prawler74** [comment]
Hi

How did you manage to save the new reziced loading area? I try to customize the same trailer, but there is no way to save it. The customize button is greyed out?
Am i missing something?

Bjørn

**Comment by MHMTALP** [comment]
Hello

Let me give you an example: Let's say there is no mudguard, I marked it as there is, then I made the adjustments and it was done

**Comment by loki79uk** [comment]
> How did you manage to save the new reziced loading area? I try to customize the same trailer, but there is no way to save it. The customize button is greyed out? Am i missing something?

You have to change something (number plate letter, wheels, etc) FIRST, that will enable the button, then adjust.  This is just a temporary workaround until the menu is working.  Or it is easier to just buy the same trailer in a different savegame since then you don't have to drive to a workshop.

**Comment by prawler74** [comment]
Thanx loki. Will give that a go....by the way. Awesome work.

Bjørn

**Comment by prawler74** [comment]
Hi

Tried to do as described. Got the customize button. But it does not apply. 

This is what the log/console says:

2024-12-14 13:46 UniversalAutoload - supported vehicle: PITTS Trailers LT40-8L #ALL (UNIQUE)
2024-12-14 13:46 *** NO LOCAL CONFIGS AVAILABLE - #ALL (UNIQUE) ***
2024-12-14 13:46 CREATE SHOP VEHICLE: PITTS Trailers LT40-8L
2024-12-14 13:46 PITTS Trailers LT40-8L: UAL SETTINGS WERE NOT ADDED
2024-12-14 13:46 findTensionBelts
2024-12-14 13:46 findLoadingSurface
2024-12-14 13:46 expandLoadingSurface
2024-12-14 13:46 SET HEIGHT
2024-12-14 13:46 INIT SHOP CONFIG
2024-12-14 13:46 DELETE SHOP VEHICLE: PITTS Trailers LT40-8L

BJørn

**Comment by loki79uk** [comment]
It should be in mod settings on the server.  Maybe your server host does not allow access to that folder?

**Comment by prawler74** [comment]
Hi again loki. Currently using VeryGames as my server host. Sent a ticket. Hope to have an answer soon.
Really wish that my settings/editing/customising would stick.
I cannot seem to make them work in singleplayer either.
Something is really strange. Do i need to roll back to an earlier version if UAL?

Bjørn

**Comment by loki79uk** [comment]
Should definitely work in SP.  What is the path to your mod settings folder?  Are there any non-ascii characters?

**Comment by prawler74** [comment]
Hi

The path is in its default location. I edited the xml file and tested it just for fun. 10 by 10 meter box on the pitts trailer. It is indeed 10 by 10 in the game, but editing it in game by changing the licenseplate and then the loading area, it still is showing up as "standard" the nest time i try to customize. Is this the correct behaviour?

Mvh
Bjørn

**Comment by loki79uk** [comment]
> Hi
> 
> The path is in its default location. I edited the xml file and tested it just for fun. 10 by 10 meter box on the pitts trailer. It is indeed 10 by 10 in the game, but editing it in game by changing the licenseplate and then the loading area, it still is showing up as "standard" the nest time i try to customize. Is this the correct behaviour?
> 
> Mvh Bjørn

No, it should overwrite the entry in the file - what does it look like?   Are there more than one?

mine looks like:
```xml
        <vehicle configFileName="data/vehicles/pittsTrailers/lt408L/lt408L.xml">
            <configuration selectedConfigs="ALL">
                <options isLogTrailer="true"/>
                <loadingArea offset="0.020 1.647 -0.257" width="2.255690" length="13.058583" height="2.609175"/>
            </configuration>
        </vehicle>
```

**Comment by prawler74** [comment]
Hi

Mine looks like this:
<?xml version="1.0" encoding="utf-8" standalone="no"?>
<universalAutoload>
    <vehicleConfigurations>
        <vehicle configFileName="data/vehicles/pittsTrailers/lt408L/lt408L.xml">
            <configuration selectedConfigs="ALL">
                <options isLogTrailer="true"/>
                <loadingArea offset="0.404 1.360 0.510" width="10" length="10" height="5"/>
            </configuration>
        </vehicle>
    </vehicleConfigurations>
</universalAutoload>

That is the only one in the modSettings folder

Bjørn

**Comment by loki79uk** [comment]
> That is the only one in the modSettings folder Openeded it in game and tried to edit the 10 by 10......but it just defaults to "standard".....

You should edit it before loading the game, otherwise the original one is already loaded into memory.

**Comment by prawler74** [comment]
Hi. 

I can see that what i just wrote can me missunderstood. 

Did it before loading into game. Got an answer on my ticket regarding the ftp access. 

Will look at it tomorrow. Currently at work. This is really strange. 

It just will not save the changes. 
Will keep you posted some time tomorrow. 

Bjørn

**Comment by loki79uk** [comment]
> Hi.
> 
> I can see that what i just wrote can me missunderstood.
> 
> Did it before loading into game. Got an answer on my ticket regarding the ftp access.
> 
> Will look at it tomorrow. Currently at work. This is really strange.
> 
> It just will not save the changes. Will keep you posted some time tomorrow.
> 
> Bjørn

Closing original bug as fixed.

@prawler74 You can still carry on this conversation here if you like, or just message me on Discord is faster.

**Comment by prawler74** [comment]
Hi

Updated to the latest version 0.2.2.0 just now.
Problem still exists. 

This is a litte info from the log.txt:
2024-12-17 12:14 frontUnloadingOnly: false
2024-12-17 12:14 Demco Steel Drop Deck: UAL - tension belts (8)
2024-12-17 12:14 SET ALL
2024-12-17 12:14 CLICKED trailerTypeListBox = false (1)
2024-12-17 12:14 CLICKED unloadingTypeListBox = false (1)
2024-12-17 12:14 UniversalAutoload - supported vehicle: Demco Steel Drop Deck #ALL (UNIQUE)
2024-12-17 12:14 *** NO LOCAL CONFIGS AVAILABLE - #ALL (UNIQUE) ***
2024-12-17 12:14 CREATE SHOP VEHICLE: Demco Steel Drop Deck
2024-12-17 12:14 Demco Steel Drop Deck: UAL SETTINGS WERE NOT ADDED
2024-12-17 12:14 findTensionBelts
2024-12-17 12:14 findLoadingSurface
2024-12-17 12:14 expandLoadingSurface
2024-12-17 12:14 SET HEIGHT
2024-12-17 12:14 SET HEIGHT
2024-12-17 12:14 INIT SHOP CONFIG
2024-12-17 12:14 CLICKED BACK
2024-12-17 12:14 DELETE SHOP VEHICLE: Demco Steel Drop Deck

Would like to chat more on Discord, ,but i need youre username. I am not that into discord and how to find users and all that.
I finally found my username and tag, boy discord is not very friendly.....Mr.Elektric#4345
Add me if you want.

Best regards
Bjørn
